packages/hub/config/structure.sql

```diff
@@ -1202,6 +1202,26 @@ CREATE TABLE public.reservations (
 
 ALTER TABLE public.reservations OWNER TO postgres;
 
+--
+-- Name: reward_proofs; Type: TABLE; Schema: public; Owner: postgres
+--
+
+CREATE TABLE public.reward_proofs (
+    reward_program_id text NOT NULL,
+    payee text NOT NULL,
+    leaf text NOT NULL,
+    payment_cycle integer NOT NULL,
```

Comment:
Not vital now but if it doesn't cause issues some futureproofing here would be to change this to a numeric(78,0) type - that covers the full range of uint256 that the underlying types used in the contract can hold.

Reply:
Not quite sure how that plays with prisma, might need to explicitly set it as a decimal(76,0) 

[@db.Decimal(p, s)†
--](https://www.prisma.io/docs/reference/api-reference/prisma-schema-reference#decimal)

Reply:
Actually - we'd hit other issues in the processing setup if we were close to this at all. Let's keep it at integer.